scripts/pkg/update-docs.go

```diff
@@ -179,13 +179,34 @@ func fixupMarkdownFile(path string) error {
 	if DryRun {
 		return nil
 	}
+
+	// Prefix link to be added before relative links in
+	// order to translate relative links into absolute links.
+	prefix := fmt.Sprintf("(https://github.com/antrea-io/antrea/blob/%s/", version)
+
+	// Translate relative links to directories build, pkg, hack, ci, CHANGELOG and LICENSE file into absolute links.
+	relativeLinksExp := regexp.MustCompile(`\([\./]*(build/|pkg/|hack/|ci/|CHANGELOG|LICENSE)+[A-Za-z0-9\.\-_/]*\)`)
```

Comment:
yes, this is pretty much what I had in mind
a couple of suggestions:
- can we move the list of file prefixes to a separate variable and build the regex using that variable? it will make maintenance easier. `const filePrefixes = []string{"build/",...}`
- is the `+` at the end of the prefix list correct?

Reply:
- Slice or const array is not supported in Go, so did a workaround by creating a function.
- No, it was a mistake, removed. Thanks!